diff_drive_controller/CMakeLists.txt

```diff
@@ -3,13 +3,14 @@ project(diff_drive_controller)
 
 find_package(catkin REQUIRED COMPONENTS
     controller_interface
-    urdf
+    nav_msgs
     realtime_tools
     tf
-    nav_msgs)
+    urdf
+)
 
 catkin_package(
-  INCLUDE_DIRS include
```

Comment:
Great insight!
I admit my approach to only change this package is a bit sloppy...but actually there's no point IMHO to expose include dirs of a controller where the "library" is only a vehicle for the plugin, not an actual library others would link against.